package/libgluonutil/src/libgluonutil.c

```diff
@@ -109,6 +109,34 @@ struct json_object * gluonutil_wrap_and_free_string(char *str) {
 	return ret;
 }
 
+
+bool gluonutil_get_mesh_prefix(struct in6_addr *result) {
+	struct json_object *site = gluonutil_load_site_config();
+	if (!site)
+		return false;
+
+	struct json_object *node_prefix = NULL;
+	if (!json_object_object_get_ex(site, "node_prefix6", &node_prefix))
+		return false;
+
+	const char *str_prefix = json_object_get_string(node_prefix);
+	if (!str_prefix)
+		return false;
```

Comment:
This error path and the one above don't `json_object_put(site)`.

Reply:
indeed, however that wasn't overlooked but actually new to the fixup. done.